src/WorldStorage/WSSAnvil.cpp

```diff
@@ -1562,6 +1562,8 @@ void cWSSAnvil::LoadEntityFromNBT(cEntityList & a_Entities, const cParsedNBT & a
 		{ "minecraft:witch",               &cWSSAnvil::LoadWitchFromNBT },
 		{ "WitherBoss",                    &cWSSAnvil::LoadWitherFromNBT },
 		{ "minecraft:wither",              &cWSSAnvil::LoadWitherFromNBT },
+		{ "WitherSkeleton",                &cWSSAnvil::LoadWitherFromNBT },
```

Comment:
This line doesn't look right.

Reply:
That explains why random withers spawned. Thanks!